docs/quick-start.md

```diff
@@ -0,0 +1,51 @@
+## Requirements
+
+- [Git](https://git-scm.com/downloads)
+- [Docker](https://www.docker.com/products/docker/)
+- [Docker Compose](https://www.docker.com/products/docker-compose)
+
+## Installation
+
+1 - Clone this repository anywhere on your machine:
+
+```bash
+git clone https://github.com/gcamrit/docker-homestead.git
+```
+2 - Now you have to configure you development environment
+```bash
+cd docker-homestead && cp .env.example .env
+```
+3 - `.env` contains some configuration you might want to adjust as your need.
+```bash
+# this is what it contains by default feel free to change as you need
+
+DOCKER_USER=homestead   # you username of your local machine
+PROJECT_ROOT=~/Projects # where you put all you project on your machine
+
+MYSQL_VERSION=5.7 # available mysql version are 5.5, 5.6, 5.7
+MYSQL_DATABASE=homestead
+MYSQL_USER=homestead
+MYSQL_PASSWORD=secret
+MYSQL_ROOT_PASSWORD=root
+```
+4 - Run Containers: *(Make sure you are in the `docker-homestead` folder before running the `docker-compose` commands).*
+```bash
+docker-compose up -d
+```
+5 - As this stack contains nginx all your project configuration file can be created in `config/sites` directory.
+
+6 - You must add the "domains" for your Nginx sites to the hosts file on your machine.On Mac and Linux, this file is located at `/etc/hosts`. On Windows, it is located at  `C:\Windows\System32\drivers\etc\hosts.` The lines you add to this file will look like the following:
+```bash
+127.0.0.1  homestead.app
+```
+
```

Comment:
@gcamrit  dai It will be good to add the following content in 7th ?

7 - Example config file for laravel application

inside the config/sites directory just create something.conf and configuration be something like this

```
server {
  listen 80;
  listen [::]:80;

  server_name laravel.test;
  root /var/www/laravel/public;

  index index.php index.html index.htm index.nginx-debian.html;

  server_name _;
  
  autoindex on;

  location / {
    try_files $uri $uri/ /index.php?$query_string;
  }

  location ~ \.php$ {
    include snippets/fastcgi-php.conf;
    fastcgi_pass 0.0.0.0:9000;
  }

  location ~ /\.ht {
    deny all;
  }

  location ~ /.well-known {
    allow all;
  }
}
```
Do not forget to register the domain in /etc/hosts 

```bash
127.0.0.1  laravel.test
```